pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -443,6 +443,9 @@ type NetworkConfig struct {
 	//
 	//     > Note: This option is mutually exclusive with CIDR.
 	//
+	//     > Note: To do link-local addressing (ex. SLAAC with IPv6), set CIDR to "" and DHCP to false.
+	//     > All other options will still apply.
```

Comment:
I think this might be worded better. Technically there is nothing the user has to do for SLAAC. It is part of the IPv6 magic. I think maybe what we really have enable is making CIDR optional.